packages/shared/lib/auxiliary/icon/icons/index.ts

```diff
@@ -0,0 +1,88 @@
+export * from './arrow-left.icon'
```

Comment:
maybe this should be an svgs folder and then name the files, .svg.ts

Reply:
updated